### ISSUE TITLE
Node.js 12 actions stopping deployment

### ISSUE THREAD

**Issue by fantazzzmo** [TryGhost/action-deploy-theme#78]
Hi,

I try deploying my ghost theme and keep getting the message as shown on the screenshot:

![image](https://user-images.githubusercontent.com/51960249/208504365-a13715fa-5c4c-40ec-a202-850b8c930810.png)

Although it show as a warning, it stops the deployment.

deploy-theme.yml is as follows:

> name: Krabi Theme Deploy
> on:
>   push:
>     branches:
>       - master
>       - main
> jobs:
>   deploy:
>     runs-on: ubuntu-latest
>     steps:
>       - uses: actions/checkout@v3.2.0
>       - uses: TryGhost/action-deploy-theme@v1.6.1
>         with:
>           api-url: ${{ secrets.GHOST_ADMIN_API_URL }}
>           api-key: ${{ secrets.GHOST_ADMIN_API_KEY }}

Please advise what to do?

**Comment by BretFisher** [comment]
On mine it warns but doesn't block, and my GHA yaml looks the same as yours. Are you sure that's the reason for your failed run? I recommend checking the logs.

**Comment by igoreliasm** [comment]
@fantazzzmo can you try to change your configuration checkout to `actions/checkout@master` and try to run again?

Like this config:

```yml
name: Deploy Theme
on:
  push:
    branches:
      - master
      - main
jobs:
  deploy:
    runs-on: ubuntu-22.04
    steps:
      - uses: actions/checkout@master
      - name: Deploy Ghost Theme
        uses: TryGhost/action-deploy-theme@v1
        with:
          api-url: ${{ secrets.GHOST_ADMIN_API_URL }}
          api-key: ${{ secrets.GHOST_ADMIN_API_KEY }}
```

**Comment by fantazzzmo** [comment]
Thanks for the help guys.

I think I found what the problem is and it has nothing to do with node.js 12 (not sure why the error message says so).

The process breaks at Theme Validation - because new theme files are using features available in Ghost 5.0 and we are still using Ghost 4.x. And we cannot update to Ghost 5.0 because the current theme version does not support Ghost 5.0. Sort of an infinite loop. It looks like we'll wait for Ghost to update to 5.0 automatically on January 31 and then update the theme. Fingers crossed all goes smoothly.
